### DISCUSSION TITLE
Sharp grain boundary visualization

### DISCUSSION THREAD

**Discussion by ashishdhole** [idaholab/moose#27174] [Q&A Modules: Phase field]
Hello All,
Is there a way to make grain boundaries as sharp lines? 'BndsCalcAux' captures data 1/4 offset on either side of the actual grain boundary (where op=0.5). There there a way or option I can choose to make it look sharp in paraview?
Thank you

**Comment by jiangwen84** [comment]
I am exactly sure about your purpose. If you just want plots look nicer, you can probably draw a 0.5 contour from all the grain order parameters.

**Comment by ashishdhole** [comment]
Thank you very much. That worked like a charm.
